src/widgetastic_patternfly/__init__.py

```diff
@@ -502,6 +503,15 @@ def currently_selected(self):
         """A property to return the currently selected menu item"""
         return [self.browser.text(el) for el in self.browser.elements(self.CURRENTLY_SELECTED)]
 
+    @property
+    def all_options(self):
+        """A property to return the list of options available in the BootstrapNav"""
+        b = self.browser
+        return [
+            el.text
+            for el in b.elements(self.ITEM_LOCATOR, parent=self)
```

Comment:
Looks good, two things come to mind.

1. Needs unit tests :)  Don't hesitate to ask if you want some guidance on this.

2. Why does `currently_selected` use `self.browser.text(el)` instead of `el.text` - is there a chance that the `text` attribute isn't set on an element?

Reply:
@mshriver about `self.browser.text` I think we have a guideline
"instead of `element.text` use `browser.text(element)` (applies for all such circumstances)"

Reply:
@digitronik can you please link to where you got that? I'm also more interested in _why_ that is the guideline.  Preliminary reading suggests it may be differentiating between innerText (what user sees) and textContent (includes non-visible text), but I'm not sure yet.

Reply:
@mshriver https://github.com/RedHatQE/widgetastic.core/blob/master/docs/guidelines.rst
`When using Browser` please clear I'm also confused

Reply:
I will update the text call. I used `parent=self` as it was used https://github.com/RedHatQE/widgetastic.patternfly/blob/47b5e98ec07d0658f6e1a1d9e01812be51833c85/src/widgetastic_patternfly/__init__.py#L1031 

Does it mean that's wrong on L1031 as well?

Reply:
Passing `parent=self` is unnecessary when using `Browser` (in the guidelines Nikhil linked) because it will figure out the parent automatically.

Reply:
Right, I have no objection on that, just curious because there are other places in the file where it is used in browser calls. Can you please check?